src/mem/epoch/markable_atomic.rs

```diff
@@ -48,32 +53,53 @@ fn opt_owned_into_usize<T>(ptr: Option<Owned<T>>, mark: usize) -> usize {
     raw as usize | mark
 }
 
-impl<T> MarkedAtomic<T> {
-    /// Create a new, null atomic pointer.
+impl<T> MarkableAtomic<T> {
+    /// Create a new, null markable atomic pointer with a marker value of 0.
     #[cfg(feature = "nightly")]
-    pub const fn null() -> MarkedAtomic<T> {
-        MarkedAtomic {
+    pub const fn zero() -> MarkableAtomic<T> {
```

Comment:
This change is good - singling out `0` as a distinguished mark does not seem like a good idea. However, maybe it would be simpler to just leave this function out and make people write `MarkedAtomic::null(0)` - it is only one more character, and is probably clearer.